geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/CreateAsyncEventQueueFunction.java

```diff
@@ -130,48 +122,19 @@ public void execute(FunctionContext context) {
 
     } catch (CacheClosedException cce) {
       context.getResultSender().lastResult(new CliFunctionResult(memberId, false, null));
-
-    } catch (VirtualMachineError e) {
-      SystemFailure.initiateFailure(e);
-      throw e;
-
-    } catch (Throwable th) {
-      SystemFailure.checkFailure();
+    } catch (Exception th) {
```

Comment:
Since the catch is no longer for {{Throwable}} a better name for the variable would be 'e'